main.go

```diff
@@ -0,0 +1,273 @@
+// Command bt2disk syncs the contents of a BigTable instance (typically, a local emulator) to/from the local disk.
+package main
+
+import (
+	"context"
+	"database/sql"
+	"flag"
+	"fmt"
+	"hash/fnv"
+	"log"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/bigtable"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+func main() {
+	db := flag.String("db", "", "Target sqlite file to save to or restore from")
+	project := flag.String("project", "local", "GCP project to connect to")
+	instance := flag.String("instance", "local", "BigTable instance to connect to")
+	gcp := flag.Bool("gcp", false, "Set to 'true' to connect to real GCP instances (safeguard)")
+
+	flag.Usage = func() {
+		_, _ = fmt.Fprintf(flag.CommandLine.Output(), "Usage:  bt2disk [-d DIR] restore|save\n")
+		flag.PrintDefaults()
+	}
+	flag.Parse()
+
+	ctx := context.Background()
```

Comment:
Might well...
```suggestion
	ctx, cancel := context.WithCancel(context.Background())
	defer cancel()
```
Just so that the context passed down to any library code (like the bigtable client stuff) can be cancelled on exit.